packages/opentelemetry-core/src/platform/browser/RandomIdGenerator.ts

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {IdGenerator} from '../../trace/IdGenerator';
```

Comment:
nit: Prefer spaces like this `{ IdGenerator }`

Reply:
Thank you! I am working on it and already passed through the `npm run lint` test locally. Will make another commit soon